src/gleam/list.gleam

```diff
@@ -1361,7 +1367,8 @@ pub fn partition(
 }
 
 /// Returns all the permutations of a list.
-/// All values must be unique.
+///
+/// All list item values must be unique.
```

Comment:
If this is the case how would you do permutations when you do want duplications? I think we can remove this note.

Reply:
Yes, but not as part of this PR, I guess. ATM the note seems to be correct, see:
https://github.com/gleam-lang/stdlib/issues/345

Reply:
Seeing as that is a bug let's remove this still.